packages/account/src/Sections/Security/DeactivateAccount/deactivate-account-reason.jsx

```diff
@@ -83,38 +69,36 @@ const GeneralErrorContent = ({ message, onClick }) => (
                 {localize('OK')}
             </Button>
         </div>
-    </React.Fragment>
+    </>
 );
 
 const character_limit_no = 110;
 const max_allowed_reasons = 3;
 
-class DeactivateAccountReason extends React.Component {
-    static contextType = PlatformContext;
-
-    state = {
-        api_error_message: '',
-        is_loading: false,
-        is_account_deactivated: false,
-        is_modal_open: false,
-        reason: null,
-        details: undefined,
-        which_modal_should_render: undefined,
-        is_checkbox_disabled: false,
-        total_checkbox_checked: 0,
-        remaining_characters: character_limit_no,
-        total_accumulated_characters: 0,
-    };
-    validateFields = values => {
+const DeactivateAccountReason = ({ onBackClick, mt5_login_list, client_accounts, dxtrade_accounts_list }) => {
+    const { is_dashboard } = React.useContext(PlatformContext);
+    const [is_account_deactivated, setAccountDeactivated] = React.useState(false);
```

Comment:
Can we use the variable name as the function name for better consistency? Like `is_account_deactivated` => `setIsAccountDeactivated`. This way we can know what kind of type the state expects too.